erpnext_germany/erpnext_germany/doctype/business_trip/business_trip.py

```diff
@@ -86,7 +90,7 @@ def on_submit(self):
 		settings = frappe.get_single("Business Trip Settings")
 		for journey in self.journeys:
 			if journey.mode_of_transport == "Car (private)":
-				description = "{distance} * {mileage_allowance} von {from_place} nach {to_place} (Fahrt mit Privatauto)".format(
+				description = "{distance} km * {mileage_allowance} von {from_place} nach {to_place} (Fahrt mit Privatauto)".format(
```

Comment:
Distance is not guaranteedd to be in km. In the settings and Business Trip we could also enter values in meters or miles - it is not specified. If we specify it here, we should also specify it in Business Trip and settings (`distance` -> `distance_in_km`).